feeds/pypi/main.go

```diff
@@ -58,32 +77,47 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-// PubSubMessage is the payload of a Pub/Sub event.
-type PubSubMessage struct {
-	Data []byte `json:"data"`
-}
-
 // Poll receives a message from Cloud Pub/Sub. Ideally, this will be from a
 // Cloud Scheduler trigger every `delta`.
-func Poll(ctx context.Context, m PubSubMessage) error {
+func Poll(ctx context.Context, t *pubsub.Topic) error {
 	packages, err := fetchPackages()
 	if err != nil {
 		return err
 	}
 	cutoff := time.Now().UTC().Add(-delta)
 	for _, pkg := range packages {
+		log.Println(pkg.Name(), pkg.Version())
 		if pkg.ModifiedDate.Before(cutoff) {
```

Comment:
```suggestion
		if pkg.ModifiedDate.After(cutoff) {
```

This was another bug I spotted during testing. Figured we could just make it part of this PR if you'd like.